config/application.rb

```diff
@@ -23,6 +23,7 @@ class Application < Rails::Application
     # -- all .rb files in that directory are automatically loaded.
 
     config.autoload_paths << Rails.root.join("app", "uploaders")
+    config.autoload_paths << Rails.root.join("app", "decorators")
```

Comment:
I don't believe that it's necessary to add `decorators` or `uploaders` to `autoload_paths` if they're sub-folders of `app`. 🤔